server/src/main/java/org/opensearch/client/node/NodeClient.java

```diff
@@ -137,6 +141,7 @@ public String getLocalNodeId() {
     private <Request extends ActionRequest, Response extends ActionResponse> TransportAction<Request, Response> transportAction(
         ActionType<Response> action
     ) {
+        /* TODO add support to read actionsRegistry along with actions */
```

Comment:
Do we have an issue to track this TODO?

Reply:
Haha I've taken care of this as part of [SDK#106](https://github.com/opensearch-project/opensearch-sdk-java/issues/107).